extended-statistics.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 扩展统计信息简介
+summary: 扩展统计信息用于优化包含相关列的查询
+---
+
+# 扩展统计信息简介
+
+[统计信息简介](/statistics.md)章节中提到的统计信息，包括直方图和 Count-Min Sketch 等，都是常规统计信息。在每次手动或自动收集统计信息时，对应表的这些常规信息都会被收集。与常规统计信息相对的另一类统计信息是扩展统计信息，只对特定场景下的优化器估算有作用。
+
+由于只在特定场景有益，扩展统计信息在默认手动或自动 `ANALYZE` 时不会被收集，避免增加收集统计信息的开销。如果想要收集扩展统计信息，需要先通过 SQL 命令进行“注册”，然后在下一次手动或自动 `ANALYZE` 时，TiDB 会在常规统计信息基础上额外收集这些注册过的扩展统计信息。
+
+## 扩展统计信息注册
+
+如果要对想收集的扩展统计信息进行注册，可使用 `CREATE STATISTICS` SQL 语句，其语法如下：
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE STATISTICS stats_name(stats_type) ON table_name(column_name, column_name...);
```

Comment:
If the statistic is created in `db1`, but the table is in `db2`, to drop the statistics, which database should the user use, `db1` or `db2`?

Reply:
`db2`.